en_US.ini

```diff
@@ -135,7 +135,7 @@ Log Console = &Log Console
 Memory View... = Memory &View...
 More Settings... = &More Settings...
 Nearest = &Nearest
-Non-Buffered Rendering = &Non-Buffered Rendering (speedhack)
+Non-Buffered Rendering = &Discard Effects (Non-Buffered, speedup)
```

Comment:
True because I wanted "simplified" to sound like "dumbed down" but I guess it didn't end up sounding like that. Meh.